test/distributed/test_dist_neighbor_sampler.py

```diff
@@ -396,21 +401,10 @@ def test_dist_neighbor_sampler(disjoint):
         s.bind(('', 0))
         port = s.getsockname()[1]
 
-    world_size = 2
-    w0 = mp_context.Process(
-        target=dist_neighbor_sampler,
-        args=(world_size, 0, port, disjoint),
-    )
-
-    w1 = mp_context.Process(
-        target=dist_neighbor_sampler,
-        args=(world_size, 1, port, disjoint),
-    )
+    procs = (ProcArgs(target=dist_neighbor_sampler, args=(0, port, disjoint)),
```

Comment:
is there a reason for using tuple and list interchangeably?

Reply:
no special reason, `assert_run_mproc`  accepts Collection as the parameter `pargs: Collection[ProcArgs],` dist_utils.py:59